<launch>
  <node name="velodyne" pkg="velodyne" type="velodyne_node" output="screen" respawn="true">
    <rosparam command="load" file="$(find velodyne)/etc/velodyne64.yaml"/>
    <param name="sensor/calibration_file" value="$(find velodyne)/etc/calib-HDL-64E.dat"/>
  </node>
</launch>
